src/runtime/runtime-simd.cc

```diff
@@ -329,7 +329,6 @@ SIMD128_UNARY_FUNCTIONS(DECLARE_SIMD_UNARY_FUNCTION)
 
 template<typename T1, typename T2>
 inline void BitsTo(T1 s, T2* t) {
```

Comment:
This doesn't look good to me if it only exists for compilation. You need to modify the macro to pass it.